server/@types/forms/index.d.ts

```diff
@@ -35,5 +35,6 @@ declare module 'forms' {
       | 'MORE_THAN_TWELVE_MONTHS'
     stepNumber: number
     status: 'NOT_STARTED' | 'ACTIVE' | 'COMPLETE'
+    action?: `delete-step-[${string}]`
```

Comment:
Not sure about this? - not sure about the string interpolation .... 🤔

Reply:
I'm not sure this is the form we want to use (by adding an action field)

I think the action "delete-step" belongs on the action field of `UpdateGoalForm`
That does then beg the question about how we tell it which step we want to remove .... let me give that some thought .... 🤔

Reply:
Yes; remove the `action` from the `UpdateStepForm` and add it to line 25 instead (ie. add to the `action` property of `UpdateGoalForm`
```
action?: 'add-another-step' | 'submit-form' | 'delete-step-[0]'
```
Hardcode it here as a zero - I dont think it actually matters at runtime, and it serves as an example that an index is passed in and it is zero indexed